api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -142,6 +142,12 @@ message OAuth2Config {
   // Automatic access token refresh will be performed for these requests, if enabled.
   // This behavior can be useful for AJAX requests.
   repeated config.route.v3.HeaderMatcher deny_redirect_matcher = 14;
+
+  // The default lifetime in seconds of the refresh token, if the exp (expiration time) claim is omitted in the refresh token or the refresh token is not JWT.
```

Comment:
I suggest to mention that this setting is only considered if _use_refresh_token_ is set to true, otherwise the authorization server expiration or _defaul_expires_in_ is used

Reply:
thank you, I will add